parsers/julia.c

```diff
@@ -1148,36 +1205,46 @@ static void parseModule (lexerState *lexer, vString *scope, int parent_kind)
 /* Import format:
  * [ "import" | "using" ] <ident> [: <name>]
  */
-static void parseImport (lexerState *lexer, vString *scope, int parent_kind)
+static void parseImport (lexerState *lexer, vString *scope, int parent_kind, int token_type)
 {
     vString *name = vStringNew();
-
+    int library_role;
+    int unknown_role;
+    if (token_type == TOKEN_IMPORT)
+    {
+        library_role = JULIA_LIBRARY_IMPORTED;
+        unknown_role = JULIA_UNKNOWN_IMPORTED;
+    }
+    else /* if (token_type) == TOKEN_USING */
+    {
+        library_role = JULIA_LIBRARY_USING;
+        unknown_role = JULIA_UNKNOWN_USING;
+    }
     /* capture the imported name */
     advanceToken(lexer, true);
+    vStringCopy(name, lexer->token_str);
+    addReferenceTag(name, K_LIBRARY, library_role, lexer->line, lexer->pos, NULL);
     if (lexer->cur_c == ':')
     {
-        advanceAndStoreChar(lexer);
-        vStringCopy(name, lexer->token_str);
+        advanceChar(lexer);
         advanceToken(lexer, true);
-    }
-
-    while (lexer->cur_token == TOKEN_IDENTIFIER || lexer->cur_token == TOKEN_MACROCALL)
-    {
-        // addTag(lexer->token_str, vStringValue(name), NULL, K_IMPORT, lexer->line, lexer->pos, scope, parent_kind);
-
-        skipWhitespace(lexer, false);
-        if (lexer->cur_c == ',')
+        while (lexer->cur_token == TOKEN_IDENTIFIER || lexer->cur_token == TOKEN_MACROCALL)
```

Comment:
It seems a tab char may be used here. M-x untabify may be needed.

Reply:
OK. Didn't know `untabify` before. Actually I ran `c-guess` on this file and seems it didn't set up the correct style for me.

Reply:
As far as I can remember, it doesn't distinguish tab and space.